htdocs/fourn/facture/contact.php

```diff
@@ -174,71 +175,83 @@
 		dol_banner_tab($object, 'ref', $linkback, 1, 'ref', 'ref', $morehtmlref);
 
 		print '<div class="fichecenter">';
+		print '<div class="fichehalfleft">';
 		print '<div class="underbanner clearboth"></div>';
 
-		print '<table class="border centpercent tableforfield">';
 
-		// Type
-		print '<tr><td class="titlefield">'.$langs->trans('Type').'</td><td colspan="4">';
+		print '<table class="border centpercent tableforfield">';
+		print '<tr><td class="titlefield">'.$langs->trans('Type').'</td><td>';
 		print $object->getLibType();
-		if ($object->type == FactureFournisseur::TYPE_REPLACEMENT) {
+		if ($object->type == FactureFournisseur::TYPE_REPLACEMENT)
+		{
 			$facreplaced = new FactureFournisseur($db);
 			$facreplaced->fetch($object->fk_facture_source);
 			print ' ('.$langs->transnoentities("ReplaceInvoice", $facreplaced->getNomUrl(1)).')';
 		}
-		if ($object->type == FactureFournisseur::TYPE_CREDIT_NOTE) {
+		if ($object->type == FactureFournisseur::TYPE_CREDIT_NOTE)
+		{
```

Comment:
Must use the PHPCS style sheet defined in  dev/setup/codesniffer/ruleset.xml 
{ must be on same line for a if (PSR 12)